bindings/ergo-lib-ios/Sources/ErgoLibC/shim.h

```diff
@@ -1,6 +1,11 @@
 #ifndef ERGO_LIB_SHIM_H_
 #define ERGO_LIB_SHIM_H_
 
+// TODO: move this to ios build config
+#ifndef ERGO_REST
```

Comment:
Bit of a hack, this should somehow be defined in the ios build config

Having no ios experience and no hardware I'm not able to figure out how to do this

Reply:
I have no idea for a better place for this `define`. Let's leave it here for now.